src/components/MessageBubble.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../styles/theme.style.js';
+import { StyleSheet, Text, View, Image, TouchableOpacity, TextInputComponent, } from 'react-native';
+import TextInputContainer from '../containers/TextInputContainer';
+import { TextInput } from 'react-native-gesture-handler';
+
+
+class MessageBubble extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        isAuthor: true,
+        isSelectable: true,
+      };
+    }
+  
+    render() {
+      return (
+        <MessageBubbleContainer isAuthor={this.state.isAuthor} backgroundColor={this.props.labelColor} isSelectable={this.props.isSelectable}>
+           <WrittenMessage
+           placeholder='Message'
+       placeholderTextColor={"#D8D8D8"}
+       multiline={true}
+     />
+          <TextInputContainer placeholder='' isConfirmed={false}>
+          </TextInputContainer>
+        </MessageBubbleContainer>
+      ); 
+    }
+  }
+  
+const MessageBubbleContainer = styled.View`
+background-color: ${props => props.isAuthor ? theme.COLOR_ORANGE : (props.backgroundColor ? props.backgroundColor :  theme. COLOR_LIGHT_GRAY)};
+border-radius: 12;
+padding-vertical: 10;
+padding-horizontal: 15;
+`;
+
+const WrittenMessage = styled.TextInput`
+color:${theme.COLOR_WHITE};
```

Comment:
Can you change the font colour to black when isAuthor is false? White on a grey background isn't very clear!

Reply:
Like in the prototype, for the alignments, maybe when isAuthor = true, shift the box to the right, and when isAuthor = false, shift the container to the left
and for the width, make it smaller!
<img width="301" alt="Screen Shot 2021-12-06 at 10 47 00 PM" src="https://user-images.githubusercontent.com/72771523/144962579-43719d7c-9c1e-4a9d-b6ba-9d9ebdfd5675.png">

Reply:
shifting the message bubble is part of another task and for this task it is assumed that the author is the current user which means isAuthor=true

Reply:
Ok cool but you can still change the width

Reply:
Following on Yas' comment, here's my feedback:
- Add a max-width so that the bubble always respects the text's width if it's shorter than the max-width
  - Max-width is needed in order to not have it span the whole conversation box
- Change the text color to theme.COLOR_BLACK if isAuthor = false